churchland_pipeline_python/equipment.py

```diff
@@ -24,6 +36,7 @@ class EquipmentCategory(dj.Lookup):
         ['task controller']
     ]
 
+
 @schema
 class Parameter(dj.Lookup):
```

Comment:
Is this the parameter used in the equipments? The name is a bit too general. Some comments on the table would be helpful.